jspwiki-main/src/main/java/org/apache/wiki/plugin/PageViewPlugin.java

```diff
@@ -446,6 +468,8 @@ public String execute( final Context context, final Map< String, String > params
                         // let the engine render the list
                         result = engine.getManager( RenderingManager.class ).textToHTML( context, buf.toString() );
```

Comment:
I guess this is one of the cases that could not be handled by `Synchronizer.synchronize`? b/c of `result` not being effectively final? Would ensuring result gets assigned only inside the lambda work? Something like

```
final String result;
Synchronizer.synchronize( lock, () -> {
    [...]
    if( show == null || STR_NONE.equals( show ) ) {
        result =STR_EMPTY;
   } else if( PARAM_COUNT.equals( show ) ) {
       [...]
        result = counter.toString();
    } else if( body != null && 0 < body.length() && STR_LIST.equals( show ) ) {
        [...]
        result = engine.getManager( RenderingManager.class ).textToHTML( context, buf.toString() );
    }
} );
```
If not it's ok to leave the code as it is, having to force the code into `Synchronized.synchronized` for the sake of doing it makes no sense